### ISSUE TITLE
step by step installation guide

### ISSUE THREAD

**Issue by githubxnoob** [monicahq/monica#4351]
Hi, can someone create a noob friendly step by step guide on how to run this locally on my windows 10 pc?

i installed docker software from the official site. i am struck now. i tried to follow some youtube tutorial on how to install a docker image. its all overwhelming at first. i messed up something so i had to reinstall docker software as a whole. 

please consider writing a noob friendly installation guide for windows 10 users.

**Comment by Zen-glen** [comment]
This is a web application, so you will need to be comfortable setting up a web server. XAMMP is beginner friendly.

I recommend you do not use Docker for this purpose on Windows 10, as you will also need to enable WSL2 and it will just get messy. It's a great set up if you're a developer but just to run a single website, it's a lot to learn.

**Comment by githubxnoob** [comment]
there is no guide for installing monica with XAMMP either.  :(

**Comment by magoun** [comment]
I don't have much experience with docker, but using homestead with virtualbox has been pretty successful for me. You can follow the instructions for setting this up in the contribution guide for developers, if you are only interested in running the application locally.

https://github.com/monicahq/monica/blob/master/docs/contribute/readme.md#install-monica-locally

**Comment by liutiming** [comment]
I am using WSL2 and docker. The whole set-up is quite easy but you will need to configure mysql in WSL2 to give user `homestead` the privilege to the database. See https://github.com/monicahq/monica/issues/4241

**Comment by zeezhao** [comment]
In case it helps someone, I was able to get monica 2.19.1 to install on Windows 10. But I followed the Generic Linux instructions :)

https://github.com/monicahq/monica/blob/master/docs/installation/providers/generic.md

I was already using xampp with php 7.4.15. I had to make changes to php.ini to enable some extensions for monica to install properly.

Key steps were:
git clone https://github.com/monicahq/monica.git
cd monica
git checkout tags/v2.19.1

//then make copy of an old .env
//I had created mysql database manually and put entries in .env

composer update

composer install --no-interaction --no-dev //I may have run a number of times as I fixed php.ini issues

php artisan monica:update --force //I ran this because I had an old database

Note: 
I may have missed out a step as I had an old installation which I tried to upgrade, before deciding to start afresh but using old database....

Anyway, it works well. The only issue is the daily "stay in touch" reminder emails not working even when I run cron command manually i.e.
php artisan schedule:run

Even though I was able to test emails using:  php artisan monica:test-email

Will resolve that when critical...

**Comment by bbrendon** [comment]
I'd like to add that out of all the self-hosted apps I have, this one is the most problematic. The author makes changes to how its deployed or how emails get scheduled or the container types. It's a moving target. Simple docker-compose pull/up often does not work.

...and of course because of these changes the docs get out dated.

**Comment by PorterRyan** [comment]
Even the Docker installation instructions should be rewritten. They aren't clear at all.

**Comment by KhaaL** [comment]
I agree with OP, even as a rather "technical" person this has given me a headache to get up and running. I might suggest changes to the docs + make a gist on how to install it with docker **IF** I get it up and running myself.
